src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/9858e22e8a.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+  <title>Document</title>
+</head>
+<body>
+   <main class="main-container">
+     <div class="to-do-container">
+       <h3>Today's To Do <span><i class="fas fa-sync-alt"></i></span></h3>
+       <p><i>Add to your list...</i></p><br>
+       <ul>
+         <li class="list-item">
+          <input type="checkbox" id="Sports" name="Sports" value="Sports">
+          <label for="Sports" id="item"> Sports <span><i class="fas fa-ellipsis-v"></i></span></label><br>
+         </li>
+
+         <li class="list-items">
+          <input type="checkbox" id="learnJavaScript" name="learnJavaScript" value="Learn JavaScript">
+          <label for="learnJavaScript" id="items"> Learn JavaScript<span><i class="fas fa-ellipsis-v"></i></span></label><br>
+         </li>
+       </ul>
```

Comment:
- [ ] **Required:** This was nicely done. However, you are required to dynamically load the list using JS. The project and module are thus arranged to get you up to speed with the latest JS-ES6 syntax.
- [ ] **Required:** At the moment, the list items have a checkbox and description. The list should alongside these have indexes.
- [ ] **Required:** The list items should be loaded in the order of their indexes.